package.json

```diff
@@ -8,33 +8,33 @@
   },
   "scripts": {
     "build": "run-s clean generate:* compile:*",
-    "clean": "rimraf dist",
     "clean-cache": "node bin/clean-cache.js",
+    "clean": "rimraf dist",
     "compile:ts": "tsc -p tsconfig.app.json",
     "config-validator": "node -r ts-node/register/transpile-only -- lib/config-validator.ts",
-    "doc-fix": "run-s markdown-lint-fix prettier-fix",
-    "generate": "run-s generate:*",
-    "generate:imports": "node tmp/tools/generate-imports.js",
     "create-json-schema": "node -r ts-node/register/transpile-only -- bin/create-json-schema.js && prettier --write \"renovate-schema.json\"",
     "debug": "node --inspect-brk -r ts-node/register/transpile-only  -- lib/renovate.ts",
-    "eslint": "eslint --ext .js,.mjs,.ts lib/ test/ tools/",
+    "doc-fix": "run-s markdown-lint-fix prettier-fix",
     "eslint-fix": "eslint --ext .js,.mjs,.ts --fix lib/ test/ tools/",
-    "jest": "cross-env NODE_ENV=test LOG_LEVEL=fatal node --expose-gc node_modules/jest/bin/jest.js",
+    "eslint": "eslint --ext .js,.mjs,.ts lib/ test/ tools/",
+    "generate:imports": "node tmp/tools/generate-imports.js",
+    "generate": "run-s generate:*",
     "jest-debug": "cross-env NODE_ENV=test LOG_LEVEL=fatal node --expose-gc --inspect-brk node_modules/jest/bin/jest.js",
     "jest-silent": "cross-env NODE_ENV=test yarn jest --reporters jest-silent-reporter",
-    "lint": "run-s ls-lint eslint prettier markdown-lint",
+    "jest": "cross-env NODE_ENV=test LOG_LEVEL=fatal node --expose-gc node_modules/jest/bin/jest.js",
```

Comment:
🤔 Doesn't need a shorter string come before a longer one?

Reply:
Do you want the sorting to be like this then instead?

From top to bottom:

1. `jest`
2. `jest-a`
3. `jest-b`

Reply:
Yes, this is what i understand as `Alphabetize` 🙃